functions/spanner/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START spanner_functions_quickstart]
+// Imports the Google Cloud client library
+const Spanner = require('@google-cloud/spanner');
+
+// Your Google Cloud Platform project ID
+const projectId = 'YOUR_PROJECT_ID';
```

Comment:
Users should not be hard-coding `projectId` variables inside Cloud Functions (the environment provides the project ID. Remove this line, and don't pass project ID into the client library constructor.